File: 241009/소수 판별기/prime-number-discriminator.js
const fs = require("fs");
const n = Number(fs.readFileSync(0).toString().trim());

let state = 0;

for (let i = 2; i <= n; i++) {
    if (i != n && n % i === 0) { //i가 n이 아니면서 n을 나누어 떨어지게 만드는 경우
        state = 1; //소수가 아님 (소수가 아닌 경우를 1)
    }
}

if (state === 1) {
    console.log("C");
} else {
    console.log("P");
}

File: 241009/소수 판별기/prime-number-discriminator.py
n = int(input())

state = 0

for i in range(n + 1):
    if (i != 1 and i != n and i % n == 0): # 1과 자기 자신이 아닌 다른 수를 약수로 갖는 경우
        state = 0 # 소수가 아니다

if state == 0: #소수이면
    print("P") #P
else:
    print("C")